src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/App.xaml

```diff
@@ -1,7 +1,8 @@
 ﻿<Application x:Class="Uno.Material.Samples.App"
 			 xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 			 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-			 xmlns:material="using:Uno.Material">
+			 xmlns:material="using:Uno.Material"
+			 RequestedTheme="Light">
```

Comment:
would that force light them? Not sure we want that!

Reply:
Debug code, forgot to remove :)